lms/templates/main_django.html

```diff
@@ -27,30 +27,33 @@
 </head>
 
 <body class="{% block bodyclass %}{% endblock %} lang_{{LANGUAGE_CODE}}">
```

Comment:
Man, github has been screwing me with the diffs lately. I added "window-wrap" around the page content.